operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -617,3 +617,19 @@ mimir.siToBytes takes 1 argument
         {{- .value }}
     {{- end -}}
 {{- end -}}
+
+{{/*
+parseCPU is used to convert Kubernetes CPU units to the corresponding float value of CPU cores.
+The returned value is a string representation. If you need to do any math on it, please
+
+mimir.parseCPU takes 1 argument
+  .value = the Kubernetes CPU request value
+*/}}
+{{- define "mimir.parseCPU" -}}
+    {{- $value_string := .value | toString -}}
+    {{- if (hasSuffix "m" $value_string) -}}
+        {{ trimSuffix "m" $value_string | float64 | mulf 0.001 -}}
```

Comment:
Doesn't this return a float? The function description says only a string is returned

Reply:
To my very limited understanding Helm, the output of any template is a string by definition, no? Basically the value is stringified automatically when this template is evaluated.

Reply:
TIL